jsonapi/src/main/java/com/alecgorge/minecraft/jsonapi/packets/netty/NettyInjector.java

```diff
@@ -79,9 +80,10 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
             };
             
             // Get the current NetworkMananger list
-            networkManagers = (List<Object>) FuzzyReflection.fromObject(serverConnection, true).
-                invokeMethod(null, "getNetworkManagers", List.class, serverConnection);
-            
+            //networkManagers = (List<Object>) FuzzyReflection.fromObject(serverConnection, true).
+            //    invokeMethod(null, "getNetworkManagers", List.class, serverConnection);
```

Comment:
why was this commented out?

my recollection was that this needs to be captured so that we don't interfere with the processing of packets for normal gameplay.

Reply:
Nice catch! I forgot that I had commented that out. I need to check it. Thanks!

It was raising a runtime exception due to not recognizing the getNetworkManagers method signature (maybe something to do with the 1.15.2 nms) and the method would not go until the end and jsonapi would break somewhere else. I commented it out just to see how far it went and it would run ok. Meanwhile I went full throttle on the build simplification and completely forgot about this. I'll have to recheck it.